### ISSUE TITLE
rawprogram*.xml and patch*.xml files required

### ISSUE THREAD

**Issue by ZekeLandon** [RaghuVarma331/scripts#1]
Hello @RaghuVarma331 

I tried to install custom ROM in Moto G60 during which accidentally locked the bootloader which inturn erased all the partitions.
As of now, my device is kind of bricked.

To restore, I'm in need of rawprogram*.xml and patch*.xml for the hanoip device.
Could you please share the same?

Thanks

**Comment by RaghuVarma331** [comment]
Hi ,

Custom rom installation will never lock your bootloader.

May be you did something wrong.

Thank you!

**Comment by mister-potter** [comment]
@RaghuVarma331 
That's correct, ROM didn't do anything. Apologies as I didn't phrase the sentence properly.

I accidentally locked the bootloader by running `fastboot flashing lock`.

Could you please help me recover the device by sharing the above mentioned files?

**Comment by ZekeLandon** [comment]
@RaghuVarma331 
That's correct, ROM didn't do anything. Apologies as I didn't phrase the sentence properly.
I accidentally locked the bootloader by running `fastboot flashing lock.`

Could you please help me recover the device by sharing the above mentioned files?
